packages/javascript/bh-shared-ui/src/views/PrivilegeZones/Certification/FilterDialog.tsx

```diff
@@ -0,0 +1,321 @@
+                                                    </SelectContent>
+                                                </SelectPortal>
+                                                <Button
+                                                    variant={'text'}
+                                                    disabled={!field.value}
+                                                    className={cn('w-1/12 p-0', { invisible: !field.value })}
+                                                    onClick={() => {
+                                                        form.setValue(field.name, '');
+                                                    }}>
+                                                    <FontAwesomeIcon icon={faClose} />
+                                                </Button>
+                                            </div>
+                                        </Select>
+                                    </FormItem>
+                                )}
+                            />
+                            <FormField
+                                name='approvedBy'
+                                control={form.control}
+                                render={({ field }) => (
+                                    <FormItem>
+                                        <FormLabel>Approved By</FormLabel>
+                                        <Select onValueChange={field.onChange} value={field.value}>
+                                            <div className='flex gap-2'>
+                                                <FormControl>
+                                                    {bloodHoundUsersQuery.isLoading ? (
+                                                        <Skeleton className='h-10 w-24' />
+                                                    ) : (
+                                                        <SelectTrigger>
+                                                            <SelectValue placeholder='Select User' />
+                                                        </SelectTrigger>
+                                                    )}
+                                                </FormControl>
+                                                <SelectPortal>
+                                                    <SelectContent>
+                                                        {bloodHoundUsersQuery.data?.map((user) => {
+                                                            if (!user.email_address) return null;
+                                                            return (
+                                                                <SelectItem
+                                                                    key={user.id}
+                                                                    value={user.email_address || user.id}>
+                                                                    {user.email_address || user.principal_name}
```

Comment:
I think we can safely remove the `|| user.principal_name` fallback because we have the check above!